dom/src/main/scala-3/fs2/dom/Dom.scala

```diff
@@ -192,13 +210,19 @@ opaque type HtmlSourceElement[F[_]] <: HtmlElement[F] = dom.HTMLSourceElement
 opaque type HtmlSpanElement[F[_]] <: HtmlElement[F] = dom.HTMLSpanElement
 opaque type HtmlStyleElement[F[_]] <: HtmlElement[F] = dom.HTMLStyleElement
 opaque type HtmlTableElement[F[_]] <: HtmlElement[F] = dom.HTMLTableElement
-opaque type HtmlTableCaptionElement[F[_]] <: HtmlElement[F] = dom.HTMLTableCaptionElement
-opaque type HtmlTableCellElement[F[_]] <: HtmlElement[F] = dom.HTMLTableCellElement
-opaque type HtmlTableColElement[F[_]] <: HtmlElement[F] = dom.HTMLTableColElement
-opaque type HtmlTableRowElement[F[_]] <: HtmlElement[F] = dom.HTMLTableRowElement
-opaque type HtmlTableSectionElement[F[_]] <: HtmlElement[F] = dom.HTMLTableSectionElement
-
-opaque type HtmlTextAreaElement[F[_]] <: HtmlElement[F] = dom.HTMLTextAreaElement
+opaque type HtmlTableCaptionElement[F[_]] <: HtmlElement[F] =
```

Comment:
Apologies I'm not why, but `scalafmt` seems to be formatting these lines.

Reply:
No problem. But how are you running scalafmt? I was also unable to reproduce this issue 😅

Reply:
First time I ran it I think I did something like `scalafmt ~/wd2 ...` from a different directory. I suspect it picked up the wrong `scala.conf` which had line limits.